app/src/main/java/agency/five/codebase/android/movieapp/ui/favorites/FavoritesScreen.kt

```diff
@@ -0,0 +1,99 @@
+package agency.five.codebase.android.movieapp.ui.favorites
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.ui.component.MovieCard
+import agency.five.codebase.android.movieapp.ui.favorites.mapper.FavoritesMapper
+import agency.five.codebase.android.movieapp.ui.favorites.mapper.FavoritesMapperImpl
+import agency.five.codebase.android.movieapp.ui.theme.MovieAppTheme
+import agency.five.codebase.android.movieapp.ui.theme.spacing
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+
+private val favoritesMapper: FavoritesMapper = FavoritesMapperImpl()
+
+val favoritesViewState = favoritesMapper.toFavoritesViewState(MoviesMock.getMoviesList())
+
+const val columnCount = 3
```

Comment:
reduce visibility as much as possible, only this screen needs this so it should be `private`, also it is a convention that constants are named with capital letters and snake case (like you did in UserScoreProgressBar).